libs/ngxtension/to-observable-signal/src/to-observable-signal.ts

```diff
@@ -6,30 +6,40 @@ import {
 } from '@angular/core';
 import {
 	toObservable,
+	toSignal,
 	type ToObservableOptions,
 } from '@angular/core/rxjs-interop';
-import type { Observable } from 'rxjs';
+import type { Observable, Subscribable } from 'rxjs';
 
 export type ObservableSignal<T> = Signal<T> & Observable<T>;
 
 export function toObservableSignal<T>(
-	s: WritableSignal<T>,
+	s: WritableSignal<T> | Observable<T> | Subscribable<T>,
```

Comment:
nit: I think `Subscribable` already includes `Observable` so we shouldn't need both in the union here.